src/sdk/object_sdk.js

```diff
@@ -160,6 +160,15 @@ class ObjectSDK {
         return bucket.namespace;
     }
 
+    async read_force_md5_etag_for_bucket(name) {
+        const { bucket } = await bucket_namespace_cache.get_with_cache({ sdk: this, name });
+        return bucket.force_md5_etag;
+    }
+
+    async read_force_md5_etag_for_account() {
```

Comment:
There is no need to make this function `async`, it only accesses memory

Reply:
yes